components/tekton-ci/base/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
 resources:
 - namespace.yaml
 - serviceaccount.yaml
-- repository.yaml
```

Comment:
Why did you removed all repositories and not only konflux-test ?

Reply:
Because I was rushing and didn't pay attention to what else was in that file.  Let's try that again